serde-avro/src/header.rs

```diff
@@ -1,9 +1,28 @@
 use serde;
-use std::collections;
+use std::{collections};
```

Comment:
Curious change!

Reply:
Yes, this is.  I must have added something here and removed it later.  I will get rid of the extra {}.